api/envoy/config/core/v3/substitution_format_string.proto

```diff
@@ -65,4 +65,14 @@ message SubstitutionFormatString {
   //   empty string, so that empty values are omitted entirely.
   // * for ``json_format`` the keys with null values are omitted in the output structure.
   bool omit_empty_values = 3;
+
+  // Specify a content_type for text_format. This will be ignored for json_format.
+  // Currently supported content-type value includes ``text/html; charset=UTF-8`` and ``text/plain``
+  // with default value being ``text/plain``.
```

Comment:
I think we can avoid relying on well-known values, see below comments. Other than this, this API is fine.